.github/workflows/R-CMD-check.yaml

```diff
@@ -1,139 +1,23 @@
-# NOTE: This workflow is overkill for most R packages
-# check-standard.yaml is likely a better choice
-# usethis::use_github_action("check-standard") will install it.
+# Workflow derived from https://github.com/rstudio/shiny-workflows
 #
-# For help debugging build failures open an issue on the RStudio community with the 'github-actions' tag.
-# https://community.rstudio.com/new-topic?category=Package%20development&tags=github-actions
+# NOTE: This Shiny team GHA workflow is overkill for most R packages.
+# For most R packages it is better to use https://github.com/r-lib/actions
```

Comment:
```suggestion
# NOTE: The workflows below are designed specifically for RStudio's R packages. Other R packages should probably instead use `usethis::use_github_action_check_standard()` as a staring point
```

Reply:
Too late. I'm not adjusting this for everyone